node/test/Test/Pos/Lrc/WorkerSpec.hs

```diff
@@ -68,92 +64,19 @@ spec = giveTestsConsts $ describe "Lrc.Worker" $ modifyMaxSuccess (const 4) $ do
 -- components.
 type GroupId = Int
 
+{-
+
 -- It's copy-pasted here because we rely on the order.
-allRichmenComponents :: [Lrc.SomeRichmenComponent]
+allRichmenComponents :: HasVarSpecConfigurations => [Lrc.SomeRichmenComponent]
 allRichmenComponents =
     [ Lrc.someRichmenComponent @Lrc.RCSsc
     , Lrc.someRichmenComponent @Lrc.RCUs
     , Lrc.someRichmenComponent @Lrc.RCDlg
     ]
 
--- | We need to generate some genesis with
```

Comment:
@pva701 I'm not sure, but it seems that this code should be commented out, not deleted.
cc @neongreen

Reply:
Agreed.

Reply:
Though... no, actually not agreed.

Reply:
1. In my branch that code was deleted, not commented out.
2. I have left above a link to an old version of this file, so the code won't be lost.

Reply:
I will do CSL-1669.